### ISSUE TITLE
call for loss info/curve.

### ISSUE THREAD

**Issue by ZeroOne96** [MhLiao/TextBoxes#115]
I re-implement this model by torch. But when i train this model using Adam, loss become nan soon although i try several learning rates. So I want to ask some loss info in the training process for further re-implement.

Really thanks.
